ci-operator/step-registry/baremetal/lab/pre/dns/baremetal-lab-pre-dns-commands.sh

```diff
@@ -16,18 +16,41 @@ SSHOPTS=(-o 'ConnectTimeout=5'
 BASE_DOMAIN=$(<"${CLUSTER_PROFILE_DIR}/base_domain")
 
 # shellcheck disable=SC1090
-. <(yq e 'to_entries | .[] | (.key + "=\"" + .value + "\"")' < "${SHARED_DIR}"/external_vips.yaml)
+if [ "${ipv6_enabled:-}" == "true" ]; then
```

Comment:
external_vips.yaml is the file we should use. It is changed when we create the load_balancer, based on different cases (disconnected, ipi/upi).... There should be no reason to change this when ipv6_enabled as far as I know. 

the vips.yaml is usually copied over to external_vips.yaml.

IIRC, vips.yaml is ALWAYS used by install-config.yaml ,but DNS needs to use external_vips.yaml because in some cases we need to bind another IP (the public one) to the DNS, still using the internal one in the install-config.yaml.

Reply:
Removed the condition and added the IPv6 addresses to the external_vips.yaml.